Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Select.hs

```diff
@@ -68,6 +71,7 @@ select :: Id DPerson.Person -> Id DEstimate.Estimate -> Flow DSelectRes
 select personId estimateId = do
   now <- getCurrentTime
   estimate <- QEstimate.findById estimateId >>= fromMaybeM (EstimateDoesNotExist estimateId.getId)
+  person <- QPerson.findById personId >>= fromMaybeM (PersonDoesNotExist personId.getId)
```

Comment:
This will increase the DB usage. Why do we need pass the language to BPP?

Reply:
Get the language in SearchRequest API. Store it in searchrequest table and use it here. We will not need to make an extra DB call to person in this case.